java/gandiva/src/test/java/org/apache/arrow/gandiva/evaluator/BaseEvaluatorTest.java

```diff
@@ -40,138 +40,138 @@
 
 import io.netty.buffer.ArrowBuf;
 
-interface BaseEvaluator {
+class BaseEvaluatorTest {
 
-  void evaluate(ArrowRecordBatch recordBatch, BufferAllocator allocator) throws GandivaException;
```

Comment:
@pravindra these changes are to move support classes to be inner, so that the outer class matches the filename. So I just moved them as block, it's just the diff here looks like a lot of changes. Let me know if this is not ok. Thanks!

Reply:
lgtm. thanks for fixing this.